rules/windows/credential_access_lsass_openprocess_api.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2023/03/02"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: Lsass access events added in Elastic Endpoint 8.7."
+min_stack_version = "8.7.0"
+updated_date = "2023/03/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies access attempts to LSASS handle, this may indicate an attempt to dump credentials from Lsass memory.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Lsass Process Access via Windows API"
+note = """## Setup
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+"""
+references = ["https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1003.001/T1003.001.md"]
+risk_score = 47
+rule_id = "ff4599cb-409f-4910-a239-52e4e6f532ff"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+any where event.category == "api" and Target.process.name == "lsass.exe"
```

Comment:
Are we purposefully keeping this generic or was there intention to make the query what was originally shared from the PR description?

```
api where process.Ext.api.name in ("OpenProcess", "OpenThread") and Target.process.name == "lsass.exe"
```

Reply:
yes on purpuse to keep it generic (to not create gaps, severity is set to medium to not create panic for FPs), we can tune it later on based on reported noisy FPs.